pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/DSMServer.java

```diff
@@ -667,14 +672,16 @@ protected void setupCustomRouting(@NonNull Config cfg) {
 
         setupMiscellaneousRoutes();
 
-        setupSomaticUploadRoutes(cfg);
-
         setupSharedRoutes(kitUtil, notificationUtil, patchUtil);
 
         setupCohortTagRoutes();
 
         setupPubSubPublisherRoutes(cfg);
 
+        setupRouteGenericErrorHandlers();
```

Comment:
I needed to define these error handlers before my routes now that they're used in them.  If the pattern is leveraged more, this may need to be bumped up higher.  I needed to define them before my routes were added.